apps/bfd-model/bfd-model-rda/mappings/FissClaim.yaml

```diff
@@ -485,6 +493,8 @@ mappings:
     # TODO: Change from to claim id field once it is available in RDA API message.
```

Comment:
Isn't this available now with the 0.13 update?

Reply:
Yes and we will start using it later in PACA-752, which is coming up soon.  When we start using it we also need to truncate the db, etc..